test/lib/create-git-repo.sh

```diff
@@ -10,6 +10,7 @@
   fi
   git add .
   if ! git diff --quiet --cached; then
-    git commit -a "$amend" -m -
+    # shellcheck disable=SC2086
+    git commit -a $amend -m -
```

Comment:
I'm fine with merging this fix for now but https://www.shellcheck.net/wiki/SC2086 recommends using an array. Then we wouldn't have to add an obscure "shellcheck disable".